compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -1420,8 +1388,11 @@ impl Builder<'a, 'll, 'tcx> {
         }
     }
 
-    fn wasm_and_missing_nontrapping_fptoint(&self) -> bool {
-        self.sess().target.arch == "wasm32"
-            && !self.sess().target_features.contains(&sym::nontrapping_dash_fptoint)
+    fn fptoint_sat_broken_in_llvm(&self) -> bool {
+        match self.tcx.sess.target.arch.as_str() {
+            // FIXME - https://bugs.llvm.org/show_bug.cgi?id=50083
+            "riscv64" => true,
```

Comment:
Rather than saying this, can you please say `get_version() < (13, 0, 0)`? This will naturally lead us to removing this workaround once we bump our minimum supported version to 13.0 or later.

Reply:
Certainly! Updated now.